metadata/db.go

```diff
@@ -27,12 +27,17 @@ const (
 	dbVersion = 1
 )
 
+// DB represents a metadata database backed by a bolt
+// database. The namespace is fully namespaced and proxies
```

Comment:
`The namespace is fully namespaced` ? You mean the `database` ?

Reply:
I did mean that, but both statements are still true 😆